experiments/zrq/ansible/experiment-01.yml

```diff
@@ -0,0 +1,27 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2019, ROE (http://www.roe.ac.uk/)
+#
+#     This information is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This information is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#   </meta:licence>
+# </meta:header>
+#
+#
```

Comment:
Might be useful to add some documentation (one line could be enough) on what the experiment is setting up at some point for this.  This may be referenced in a separate file, but a short line in the config might be useful. 

Along these lines a suggestion that might make it clearer is a small change to the name to make it more meaningful?  i.e. create-cluster-experiment-01.yml